internal/cache/init_test.go

```diff
@@ -0,0 +1,14 @@
+package cache
+
+import (
+	"testing"
+
+	"github.com/edgexfoundry/device-sdk-go/internal/common"
+	"github.com/edgexfoundry/device-sdk-go/internal/mock"
+)
+
+func TestInitCache(t *testing.T) {
+	common.ValueDescriptorClient = &mock.ValueDescriptorMock{}
+	common.DeviceClient = &mock.DeviceClientMock{}
+	InitCache()
```

Comment:
add assertion to verify device, valuedescriptor, and profile in cache are expected